BeagleBone_black/src/navigation/navigation.cpp

```diff
@@ -20,32 +20,79 @@
 
 #include "navigation.hpp"
 
+#include "data/data_point.hpp"
+#include "utils/math/integrator.hpp"
+#include "utils/math/quaternion.hpp"
+#include "utils/math/vector.hpp"
+
+using hyped::data::DataPoint;
+using hyped::utils::math::Integrator;
+using hyped::utils::math::Quaternion;
+using hyped::utils::math::Vector;
+
+namespace hyped {
+namespace navigation {
+
 // Public methods
-void Navigation::update() {}
+Integrator<Vector<double, 3>> acc_to_vel;
+Integrator<Vector<double, 3>> vel_to_dis;
```

Comment:
These should be declared in `.hpp` as private members just like the `Kalman` objects.